notebooks/official/tabular_workflows/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -29,22 +29,22 @@
         "id": "title"
```

Comment:
<p>Since we are moving the notebook to a new folder, should we use the opportunity to rename it as well? Since we're dropping the comparison with Vertex AI Forecasting, should we rename to "bqml_arima_plus.ipynb"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>